db/mysql_structure_init.sql

```diff
@@ -4,6 +4,7 @@
 -- mysql -u sdrAdmin -p --default-character-set=utf8 archive_catalog_development < db/mysql_structure_init.sql
 
 SET FOREIGN_KEY_CHECKS=0;
+SET SQL_MODE='ALLOW_INVALID_DATES';
```

Comment:
Updates to MySQL and something about a "strict" mode will not allow setting a zero date as a default.  This is a work-around to allow setting a zero date (the implicit default).  If that's OK, then I can add a comment to that effect.  Otherwise, what shall we do?